lib/Sema/CSDiagnostics.h

```diff
@@ -804,6 +804,22 @@ class ContextualFailure : public FailureDiagnostic {
   }
 };
 
+class NonClassTypeToAnyObjectConversionFailure final
+    : public ContextualFailure {
+  Type RawFromType, RawToType;
+
+public:
+  NonClassTypeToAnyObjectConversionFailure(
+      const Solution &solution, Type lhs, Type rhs, ConstraintLocator *locator,
+      FixBehavior fixBehavior = FixBehavior::Error)
+      : ContextualFailure(solution, lhs, rhs, locator, fixBehavior),
```

Comment:
This fix behavior is always FixBehavior:Error so I don't think we need to provide that parameter/argument at all here.